es-ES/readme.md

```diff
@@ -0,0 +1,731 @@
+
+
+## Consejos
+
+### Los archivos temporales
+
+Ejecutar tests simultáneamente tiene algunos desafíos, hacer IO es uno. Por lo general, los test por defecto crean directorios temporales en el directorio de test actual y lo limpian al final. Esto no funcionará cuando se ejecuten tests en paralelo y entrando en conflicto entre sí. La forma correcta de hacerlo es utilizar un nuevo directorio temporal para cada test. Los módulos [`tempfile`](https://github.com/sindresorhus/tempfile) y [`temp-write`](https://github.com/sindresorhus/temp-write) pueden ser útiles.
+
+### Depuración
+
+AVA ejecuta pruebas en paralelo por defecto, que es subóptimo cuando se necesita depurar algo. En su lugar, ejecuta pruebas en serie con la opción `--serial`:
+
+```
+$ ava --serial
+```
+
+### Cobertura de código
+
+Puede usar [`istanbul`](https://github.com/gotwarlost/istanbul) para la cobertura de código como AVA [spawns the test files](#entorno-aislado), pero puede usar [`nyc`](https://github.com/bcoe/nyc) en su luegar, que es básicamente `istanbul` con soporte para subprocesos.
+
+Desde la versión `5.0.0` utiliza source maps para reportar la cobertura de su código real, independientemente de la transpilación. Asegúrese de que el código que está probando incluye un source map en línea o referencia a un archivo de source map de origen. Si usa `babel/register`se puede establecer la opción `sourceMaps` en su `.babelrc` a `inline`.
+
+
+## FAQ
+
+### Por qué no `mocha`, `tape`, `node-tap`?
+
+Mocha requiere el uso de globales implícitas como `describe` y `it` por defecto (que utilizan la mayoría de las personas), sin demasiadas opiniones buenas, hinchado, síncrono por defecto, ejecuta los tests en serie y es lento. Tape y node-tap son bastante buenos. AVA está muy inspirado en su sintaxis. Sin embargo, ambos ejecutan los tests en serie y han hecho de [TAP](https://testanything.org) un ciudadano de primera clase que ha hecho en mi humilde opinión que sus códigos base sean un poco enrevesadas y acopladas. La salida de TAP es difícil de leer, así que siempre terminan usando una herramienta de reporte de TAP externa. AVA tiene bastantes buenas opiniones y es concurrente. Viene con un simple reporter por defecto y tiene voluntad en el futuro de dar apoyo a Tun TAP reporter.
+
+### ¿Cómo es el nombre escrito y pronunciado?
+
+AVA, no Ava o ava. Pornunciado [`/ˈeɪvə/` ay-və](media/pronunciation.m4a?raw=true).
+
+### ¿Cuál es el fondo de la cabecera?
+
+[Andromeda galaxy.](https://simple.wikipedia.org/wiki/Andromeda_galaxy)
```

Comment:
There is a link to Wikipedia in Spanish. it may be better? (`https://es.wikipedia.org/wiki/Galaxia_de_Andr%C3%B3meda`)